lib/osf-components/addon/components/move-file-modal/styles.scss

```diff
@@ -59,3 +59,8 @@
 .success {
     color: $brand-success;
 }
+
+.InvalidDestination {
+    color: $color-text-gray;
```

Comment:
I would suggest increasing font weight for the invalid destination as well. $color-text-gray also gets washed a bit and this could ensure it catches the user's attention better but not necessary.